lib/partiql-isl/README.md

```diff
@@ -0,0 +1,93 @@
+## PartiQL ISL Kotlin
```

Comment:
For consistency, should the subproject still be named `partiql-isl-kotlin` rather than `partiql-isl`? The [maven](https://mvnrepository.com/artifact/org.partiql/partiql-isl-kotlin) artifact is still called `partiql-isl-kotlin`

Reply:
I didn't want to suffix kotlin on modules within a kotlin package. Seems redundant.

For example, `lang` has artifact`partiql-lang-kotlin`, so do we rename the module to `partiql-lang-kotlin`?

Reply:
I get your point. Adding the kotlin suffix is somewhat redundant since it's already within a kotlin package. Fine w/ the current naming